src/mono/wasm/debugger/tests/ApplyUpdateReferencedAssembly/MethodBody1.cs

```diff
@@ -77,6 +77,24 @@ public class MethodBody6 {
     
     
     
+    
```

Comment:
Can we replace the empty lines with an empty comment block saying that this shouldn't be changed?